tola/model_utils.py

```diff
@@ -135,7 +135,7 @@ def get_serializer(*serializer_classes):
                 purpose: (optional) purpose of this mixin (i.e. "RFAware")
                 model (required for exactly one of the composed classes): a django model
                 fields: list of fields corresponding to fields defined on the parent class or base model fields
-                override_fields: boolean - ignore all other mixins' "fields" attributes (deprecarted)
+                override_fields: boolean - ignore all other mixins' "fields" attributes (deprecated)
```

Comment:
I am seeing one use of override_fields in the IPTTQSMixin.  Is this something you were getting away from and didn't quite get rid of all the places where you were using it?